New station for everyone! [TM]

Maslin41 opened the pull request:
## О Pull Request

Ну крч, хуе мое, тыры пыры, трали сяси. Я тут наделал карту, если чет надо поправить - поправлю к вечеру.

## Как это может улучшить/повлиять на игровой процесс/ролевую игру

Вас не заебал айс -> коробка -> айс? Меня да, потому вот новая карта.

## Доказательства тестирования

Поверите на слово, что там 90% всего работает как надо? Оставшиеся 10 мб надо доработать

## Changelog

:cl:Maslin_
add: Added new station!
config: Station can be selected by a vote from 30 players.
/:cl:

Maslin41 committed:
New Statiion for everyone!

Ну крч, хуе мое, тыры пыры, трали сяси. Я тут наделал карту, потом мообщу когда в тм кинуть

Maslin41 committed:
Проебался с путем, поправил

пу-пу-пууууу

Maslin41 committed:
Проебался с путем

Пу-пу-пууууууу

Maslin41 committed:
Исправление приколов от тестера

"Починил" вроде все

Maslin41 committed:
Новая попытка исправить

Дубль 3

Maslin41 committed:
Update station.dmm

Maslin41 committed:
d

d

Kocma-san commented on _maps/station.json:
```diff
@@ -0,0 +1,17 @@
+{
+	"version": 1,
+	"map_name": "Station",
+	"map_path": "map_file/Station",
```
```suggestion
	"map_path": "map_files/Station",
```

Maslin41 committed:
dddd

dududududududududud

Maslin41 committed:
Изменения + исправления

Сосал?

Maslin41 committed:
Да

Пизда

Maslin41 committed:
Исправление зон для ящиков с заказов отделов

Киси-киси, мяу-мяу. Киси-киси, мя-мя-мяу

Maslin41 committed:
Dorms update

Добавлены дормы глав, разширены обычные дормитории, исправлены некоторые косяки.

Maslin41 committed:
Исправления обновы дорм

Косяки поправил, некоторые штуки порезал (2 дормы),  80% ламп теперь одной яркости.

Maslin41 committed:
Куст апдейт

Добавлены кусты и скамейки, ака "барикады" для сб/антагов, на прямых и длинных коридорах. Добавил пару комнат в теха.

Maslin41 committed:
Обновка +

Maslin41 committed:
Небольшие переделки

Расширена комната отдыха ГП и робототехника, переделаны декали и декор в коридорах и много чего еще.

Maslin41 committed:
Забыл про консоль для кухни

Кря

Maslin41 committed:
Merge remote-tracking branch 'upstream/master' into New_Station_for_everyone!

Maslin41 committed:
"Починил"

Обновил зоны, из-за которых карта становилась как сыр.

Maslin41 committed:
Исправление бед обнов + переделка атмоса

Исправление бед с обновлением зон + небольшое расширение атмоса.

Maslin41 committed:
Доработка исправлений

Да

Maslin41 committed:
meow

meow

Maslin41 committed:
Обновление меда

Обновлена структура медблока + исправлены мелкие косяки

Maslin41 committed:
Merge remote-tracking branch 'upstream/master' into New_Station_for_everyone!

Maslin41 committed:
МГАОВ!!

Поправил косяки с прошлой версии, прееделал РНД и мусорные трубы.

Maslin41 committed:
Микро обнова

Исправил косяк с робо + немного почистил мед + добавил спреи с чистюлей + хз

Maslin41 committed:
Обновил кухню и добавил в бриг н-секи

Киси-киси, мя-мя-мяу

Maslin41 committed:
Переработка РНД

мяу

Maslin41 committed:
Исправлен спутник ИИ и добавлены маяки для ботов

текст

Maslin41 committed:
Исправление косяков, опять

Починено РНД и переделки станции

Maslin41 committed:
Поправил 1(!) предмет

.

Maslin41 committed:
.

.

Maslin41 committed:
.

.

Maslin41 committed:
Merge branch 'New_Station_for_everyone!' of https://github.com/Maslin41/FluffySTG-Maslina_shidcode_testdorm- into New_Station_for_everyone!

Maslin41 committed:
vote

/

Maslin41 committed:
Merge remote-tracking branch 'upstream/master' into New_Station_for_everyone!

Maslin41 committed:
Правки косяков с проводкой + смена цвета труб

.

Maslin41 committed:
небольшие изменения

.

Maslin41 committed:
Merge remote-tracking branch 'upstream/master' into New_Station_for_everyone!

Maslin41 committed:
Обнова атмоса и немного робо

.

Maslin41 committed:
Merge remote-tracking branch 'upstream/master' into New_Station_for_everyone!

Maslin41 committed:
Обновление для бордера

Исправление небольших не доработок + добавлен телепорт булыги + ботики

Maslin41 committed:
мяу

Установлены провода под солярами (те что +/- целые)

Maslin41 committed:
Merge remote-tracking branch 'upstream/master' into New_Station_for_everyone!

Maslin41 committed:
Персы

Добавлены заправленные принтеры за место пустых

Maslin41 committed:
Merge remote-tracking branch 'upstream/master' into New_Station_for_everyone!
